### ISSUE TITLE
How To Make A Graded Select Boxes In A Form

### ISSUE THREAD

**Issue by marcq64** [crocodic-studio/crudbooster#1305]
crudbooster: 5.4 | laravel/framework: 5.7.* | php: 7.1.3 | 

Hi,

I tried to apply 

https://github.com/crocodic-studio/crudbooster/blob/master/docs/en/how-make-graded-select-box.md

to one of my CRUD in order to make 2 select dropdowns dependent (When value is chosen in product_types only the values of select product_names where product_names.id_product_type are equal to product_types.id are displayed , but without success :

**TABLES**
product_types
Column	Type	Comment
id	int(10) unsigned Auto Increment	 
product_type	varchar(255)	 
created_at	timestamp NULL	 
updated_at	timestamp NULL

product_names
Column	Type	Comment
id	int(10) unsigned Auto Increment	 
id_product_type	int(10) unsigned NULL	 
product_name	varchar(255)	 
created_at	timestamp NULL	 
updated_at	timestamp NULL

**SELECT TYPE**	 

$this->form[] = ['label'=>'Type','name'=>'Type','type'=>'select','validation'=>'required','width'=>'col-sm-10','datatable'=>'product_types,product_type'];

**SELECT NAME**

$this->form[] = ['label'=>'Nom','name'=>'Nom','type'=>'select','validation'=>'required','width'=>'col-sm-10','datatable'=>'product_names,product_name','parent_select'=>'product_types_id'];

Would appreciate some help here, thanks in advance, cheers, Marc

**Comment by perlbrain** [comment]
I had a lot of problems with this and I had to create a field in the parent table that had the same name which refers in 'parent_select'. Try to add a field named 'product_types_id' in the parent table and see if it works. (for the test I made 2 test tables, countries and cities with parent and see if this works).
The 'parent_select' needs the name in the parent tables. In the example https://github.com/crocodic-studio/crudbooster/blob/master/docs/en/how-make-graded-select-box.md, you see that 'provinces_id' is in both code (however not in the table, that was misleading for me).
